### PR TITLE
feat: Report mobility parking violations through Nivel API

### PR DESCRIPTION
**Pull request by KristianSelnas**
Closes https://github.com/AtB-AS/kundevendt/issues/8612

Nivel API documentation: https://drive.google.com/file/d/1ZkxN4s544oOmatZ1j3ADOQw-3FXfqgYE/view

This BFF implementation is basically just a pass through of the Nivel api, with the exception of their api key which is added by BFF

### CODE REVIEW

**Comment by gorandalum** [comment]
Could be nice to have the API doc available. Is it online or in a document somewhere?

**Comment by KristianSelnas** [comment]
I've posted a link to the Nivel api documentation. That is all I have.

File: src/api/mobility/index.ts [hunk 1]
```diff
@@ -121,4 +127,53 @@ export default (server: Hapi.Server) => (service: IMobilityService) => {
...
   });
+
+  /**
+   * Parking violations reporting
+   */
+  server.route({
+    method: 'GET',
+    path: '/bff/v2/mobility/violations-reporting/init',
+    options: {
+      tags: ['api', ' mobility', 'violations'],
```
Comment (rosvik) [inline src/api/mobility/index.ts:138]: Noticed you have a stray whitespace here, and I'm not sure how well swagger will handle that. Same with the other endpoints.

**Review by rosvik — APPROVED** [review]
LGTM 👍

File: src/api/mobility/index.ts (hunk 1 above)
Comment (marius-at-atb) [inline src/api/mobility/index.ts:138]: (whitespace still here)

File: src/api/mobility/index.ts [hunk 2]
```diff
@@ -121,4 +127,53 @@ export default (server: Hapi.Server) => (service: IMobilityService) => {
...
+        await service.initViolationsReporting(payload, h.request)
+      ).unwrap();
+    },
+  });
+
+  server.route({
+    method: 'GET',
+    path: '/bff/v2/mobility/violations-reporting/vehicle',
+    options: {
+      tags: ['api', ' mobility', 'violations', 'vehicle lookup'],
```
Comment (marius-at-atb) [inline src/api/mobility/index.ts:154]: (whitespace still here)

**Review by marius-at-atb — COMMENTED** [review]
Lgtm! Just a note that there are still two places with whitespace

**Pull request merged**